cli/ts/commands/publish.ts

```diff
@@ -0,0 +1,137 @@
+import { Keypair, PCommand, PrivKey, PubKey } from "maci-domainobjs"
+import { info, logError, logGreen } from "../utils/theme"
+import { readContractAddress } from "../utils/storage"
+import { contractExists } from "../utils/contracts"
+import { getDefaultSigner, parseArtifact } from "maci-contracts"
+import { promptPwd } from "../utils/prompts"
```

Comment:
@ctrlc03 , have you tried creating a hardhat task that imports any maci-cli command?  I'm getting error (see below).  I believe the hardhat import in the getDefaultSigner and parseArtifact is the issue.  Hadhat scripts work fine, just the hardhat `task` gives errors


```
Error HH9: Error while loading Hardhat's configuration.

You probably tried to import the "hardhat" module from your config or a file imported from it.
This is not possible, as Hardhat can't be initialized while its config is being defined.

To learn more about how to access the Hardhat Runtime Environment from different contexts go to https://hardhat.org/hre

For more info go to https://hardhat.org/HH9 or run Hardhat with --show-stack-traces```

Reply:
I have not, I will give it a try as soon as I get a chance